entry.go

```diff
@@ -87,3 +89,48 @@ func (service *EntriesService) Get(spaceID, entryID string) (*Entry, error) {
 
 	return &entry, err
 }
+
+// Delete the entry
+func (service *EntriesService) Delete(spaceID string, entryID string) error {
+	path := fmt.Sprintf("/spaces/%s/entries/%s", spaceID, entryID)
+	method := "DELETE"
+
+	req, err := service.c.newRequest(method, path, nil, nil)
+	if err != nil {
+		return err
+	}
+
+	return service.c.do(req, nil)
+}
+
+// Activate the entry, a.k.a publish
+func (service *EntriesService) Activate(spaceID string, entry *Entry) error {
```

Comment:
How about renaming this method to `Publish`. and `Unpublish` for the next one?

Reply:
Method naming is based on similar methods in 'content_type.go'.

Reply:
I still highly recommend to use `Publish` and `Unpublish`. I will change the other occurrences of `Activate` and `Deactivate`.

Reply:
ok